index.Rmd

```diff
@@ -25,7 +25,7 @@ library(scales)
 library(kableExtra)
 
 loadacs5 <- load_variables(2018, "acs5", cache = TRUE)
-#api_key <- census_api_key("type_your_API_here")
+#api_key <- census_api_key("type_your_API_here") 
```

Comment:
I think it is better to delete this line.

Reply:
Adam told me to have a line like that.